### PR TITLE
Feature/hr admin navbar

### PR DESCRIPTION
**Pull request by Abdelrhman-Samir-99**
Still need to create the onClickOut handler.

### CODE REVIEW

File: invoice-tracker-frontend/src/components/Buttons/BellNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+const BellButton = ({ count }:notificationProps ): JSX.Element => {
+
+    const [notiNumber, setNotiNumber] = useState(count);
+    const [bellColor, setBellColor] = useState("none");
+    const [isOpen, setIsOpen] = useState(false);
+    const btnRef = React.useRef<HTMLButtonElement>(null);
+    const onBellClickHandler = () => {
+        bellColor === "none" ? setBellColor("blue") : setBellColor("none");
```
Comment (MOSTA3D) [inline invoice-tracker-frontend/src/components/Buttons/BellNotification.tsx:14]: You can instead make `bellColor` as a boolean state, and use ternary operator inside `setBellColor`

File: invoice-tracker-frontend/src/components/Buttons/BellNotification.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        setNotiNumber(0);
+        setIsOpen(true);
+    }
+
+    useEffect(() => {
+        const detectOutSideClick = (e:any) => {
+     //       console.log("OUT");
```
Comment (MOSTA3D) [inline invoice-tracker-frontend/src/components/Buttons/BellNotification.tsx:21]: remove unnecessary comments

File: invoice-tracker-frontend/src/components/cegedim-logo/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import logo from '../../assets/Cegedim_Logo.jpg'
+
+const Logo = () => {
+  return (
+    <a href="#" className="flex items-center">
```
Comment (MOSTA3D) [inline invoice-tracker-frontend/src/components/cegedim-logo/Logo.tsx:6]: use React Router `Link` instead of `<a>`

**Review by MOSTA3D — APPROVED** [review]

**Comment by TalaatHarb** [comment]
If you can remove company name from the repository then that would be better and will allow me to keep the repository public after you finish.

**Review by TalaatHarb — APPROVED** [review]

**Review by hashemy21 — APPROVED** [review]

**Pull request merged**